docs/proposals/secrets-proposal.md

```diff
@@ -0,0 +1,133 @@
+## Proposal for supporting secrets in OpenCompose
+
+### Kubernetes Secrets
+
+A very simple secret object in Kubernetes looks like this -
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kubesec
+type: Opaque
+data:
+  username: <base64 encoded value>
+  password: <base64 encoded value>
+```
+
+Note that, as seen above, one secret can have multiple `data` values.
+
+---
+
+#### Creating a new secret:
+
+To create a new secret, a top level `secrets` key can be used as follows:
+
+ ```yaml
+version: "0.1-dev"
+
+services:
+   name: foobar
+   containers:
+   - image: foo/bar:tag
+     ports:
+     - port: 8080:80
+     env:
+     - foo=bar
+     - travis_creds:
```

Comment:
To be honest I think this can't stay as it is. It just creates a lot of confusion. 

I think that the way you are referencing secret here is ok, we will have to change the old `key=value` string and come with something else.

How about doing it similar to  Kubernetes?
We don't have to reinvent everything just for the sake of creating something new. Let's stay with k8s way of doing things where it makes sense.

how about this?
```
  env:
     - name: bar
        value: foo
     - name: travis_creds
        secret:  ci_secret/travis 
```